core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -12,6 +12,16 @@ cluster_groups:
   - build09
   - build10
   - vsphere02
+  managed_clusters:
+  - app.ci
+  - build01
+  - build02
+  - build03
+  - build04
+  - build05
+  - build09
+  - build10
+  - hosted-mgmt
```

Comment:
I think cluster-init should be aware of this group so that it will add the new cluster into the group while onboarding.

Reply:
Yes cluster init will be updated later